src/handler.ts

```diff
@@ -42,27 +45,43 @@ interface Matched {
  * @param projects Sentry extension projects configurations.
  * @return Sentry projectID this document reports to.
  */
-export function matchSentryProject(params: Params, projects: SentryProject[]): Matched | null {
-    if (!projects || !params.repo || !params.file) {
+export function matchSentryProject(params: Params | null, projects: SentryProject[]): Matched | null {
+    if (!projects || !params || !params.repo || !params.file) {
```

Comment:
Since this function returns `null` when `params` is null, I'd try keeping `params: Params` (required) and forcing the caller to handle that.